modules/300-prometheus/hooks/alertmanager_crd_discovery.go

```diff
@@ -38,66 +38,8 @@ var _ = sdk.RegisterFunc(&go_hook.HookConfig{
 			Kind:       "CustomAlertmanager",
 			FilterFunc: applyAlertmanagerCRDFilter,
 		},
-		{
```

Comment:
What is with services? I think we still need them. Someone can still use a labels service to send alerts.

Reply:
Alertmanager The Movie Part II - revenge of labeled services !  :smile: